onnx/backend/test/case/node/bernoulli.py

```diff
@@ -25,8 +25,8 @@ def export_bernoulli_without_dtype() -> None:
             outputs=["y"],
         )
 
-        x = np.random.uniform(0.0, 1.0, 10).astype(np.float)
-        y = bernoulli_reference_implementation(x, np.float)
+        x = np.random.uniform(0.0, 1.0, 10).astype(np.float32)
+        y = bernoulli_reference_implementation(x, np.float32)
```

Comment:
```suggestion
        y = bernoulli_reference_implementation(x, float)
```
Actually np.float was np.float64 instead of np.float32. Still, directly using float seems more clear here.

Reply:
I changed the reference implementation values dtype to `float`s.